packages/adapter-netlify/index.d.ts

```diff
@@ -1,4 +1,4 @@
 import { Adapter } from '@sveltejs/kit';
-import './ambient.js';
+import './internal.js';
```

Comment:
Why do we even have this import there? Was it needed prior to the tsconfig adjustment which loads it on its own? If this index.d.ts ends up in the public types but the internal.d.ts does not we should not reference it here

```suggestion
```